modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -92,10 +92,12 @@ include::snippets/ptp_PtpConfigBoundary.yaml[]
 |Specify `.recommend.match` rules with `nodeLabel` or `nodeName`.
 
 |`.recommend.match.nodeLabel`
-|Update `nodeLabel` with the `key` of `node.Labels` from the node object by using the `oc get nodes --show-labels` command. For example: `node-role.kubernetes.io/worker`.
+|Set `nodeLabel` with the `key` of `node.Labels` from the node object by using the `oc get nodes --show-labels` command.
+For example: `node-role.kubernetes.io/worker`.
 
-|`.recommend.match.nodeLabel`
-|Update `nodeName` with value of `node.Name` from the node object by using the `oc get nodes` command. For example: `compute-0.example.com`.
+|`.recommend.match.nodeName`
```

Comment:
I _think_ the `for example...` throughout the PR could be added to the previous sentence if you wanted.